makefiles/ddrescue.mk

```diff
@@ -0,0 +1,42 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += ddrescue
+DDRESCUE_VERSION := 1.25
+DEB_DDRESCUE_V   ?= $(DDRESCUE_VERSION)
+
+ddrescue-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) http://mirror.keystealth.org/gnu/ddrescue/ddrescue-$(DDRESCUE_VERSION).tar.lz
+	$(call EXTRACT_TAR,ddrescue-$(DDRESCUE_VERSION).tar.lz,ddrescue-$(DDRESCUE_VERSION),ddrescue)
```

Comment:
Download the package signature provided in the mirror, and add a check to the Makefile with ``PGP_VERIFY``.